client/components/gravatar/index.jsx

```diff
@@ -33,6 +42,16 @@ module.exports = React.createClass( {
 		};
 	},
 
+	componentDidMount() {
```

Comment:
Btw, good point that we should avoid unpure functions like Date.now() in the reducer, as this creates unpredictable behavior. (It's also explicitly mentioned in the (docs)[http://redux.js.org/docs/basics/Reducers.html#handling-actions]

Thinking on this some more, let's go with the simple case (not persisting) and perhaps follow up in another PR if the behavior isn't good enough.